### ISSUE TITLE
Failed to deserialize a Bitstamp asset movement - notification

### ISSUE THREAD

**Issue by maravac** [rotki/rotki#2232]
## Problem Definition

I often get tis notification in the top right corner:

Backend messages:
Failed to deserialize a Bitstamp asset movement. Check logs for details. Ignoring it.

## Logs

the log is empty

### System Description

Operating system: Win 10 Pro 64bit
Rotki version: 1.13.0

**Comment by LefterisJP** [comment]
Hello @maravac we need logs for this. Check my comment on your other issue to figure out how to give us logs with the offending asset movement.

**Comment by maravac** [comment]
I tried to open it in a debug mode via UI as well as via manual adding the .json file, as described, but still the rotkehlchen.log file is 0 bytes (empty).

**Comment by LefterisJP** [comment]
While going through another user's logs for another problem I found a bitstamp asset movements deserialization error so I am going to assume it's the same cause.

Log entry amounts/values/date changed to preserve user's privacy
```
[DATE] ERROR rotkehlchen.exchanges.bitstamp: Error processing a Bitstamp asset movement. raw_result={'fee': FVal(0.80), 'btc_usd': FVal(0.00), 'datetime': '2018-03-21 06:46:06.559877', 'usd': FVal(0.0), 'btc': FVal(0.0), 'type': 1, 'id': 123, 'eur': FVal(-100)}, error=Could not deserialize Bitstamp asset movement from user transaction. Unexpected asset amount combination found in: {'fee': FVal(0.80), 'btc_usd': FVal(0.00), 'datetime': '2018-03-21 06:46:06.559877', 'usd': FVal(0.0), 'btc': FVal(0.0), 'type': 1, 'id': 123, 'eur': FVal(-100)}.
```

Going to look at the code and their docs to see what is happening.

**Comment by maravac** [comment]
Hi,
few last days I had no time to open the Rotki app. But today I started it again and it displays another errors, including problem with Kraken exchange API key. (Ofcourse first I downloaded and installed the most recent version 1.13.1 of Rotki):

![obrazek](https://user-images.githubusercontent.com/37882327/107125395-10c4e400-68aa-11eb-9b48-28ffcc75e9a1.png)

OK, so I simply removed the Kraken API and tried to setup it again. But it is impossible not to even add the API:

![obrazek](https://user-images.githubusercontent.com/37882327/107125541-e7f11e80-68aa-11eb-93a7-e24043b067ba.png)


This is really pain. First, My Binance exchange transactions are not synchronized, then problem with Bitstamp deserialization and now even problem with Kraken API...   :)

**Comment by LefterisJP** [comment]
As I mentioned in the other issue your system clock seems to be completely out of sync. That will result in such errors querying exchange APIS and more. You can't use rotki or other applications that query the internet without a synchronized system clock. Please synchronize your system clock. Also make sure your Kraken API keys are valid.

**Comment by maravac** [comment]
No, this one is my home computer which is OK in terms och correct time synchronization. That other PC, where I had the time problem was my office PC, where I tried Rotki as well, because in my home PC I have problem to switch into debug mode. 
Btw. in my office PC I am able to switch the debug mode (even that there is tha t time sync problem), but also there the rotkehlchen.log file is empty.

My API keys are 100% valid, I did not change anything with them.

**Comment by LefterisJP** [comment]
> My API keys are 100% valid, I did not change anything with them.

It's either your keys are invalid, your clock is out of sync or Kraken is having problems. And without logs it's virtually impossible to provide any further assistance.

I found the problem of the bitstamp asset movement issue thanks to the other user's logs and will fix this issue now.

---

Now as to the reason why your logs are not there, I have no idea. We tried to help you in the other issue where @kelsos also wrote an explanation as to what may be wrong. No other Windows user has ever had problems generating logs. Let's just use https://github.com/rotki/rotki/issues/2230 to see if there is anything more we can do to figure out why your logs are not there.
